python/ql/src/semmle/python/frameworks/Django.qll

```diff
@@ -16,7 +16,284 @@ private import semmle.python.regex
  * Provides models for the `django` PyPI package.
  * See https://www.djangoproject.com/.
  */
-private module Django {
+module Django {
+  /** Provides models for the `django.views` module */
+  module Views {
+    /**
+     * Provides models for the `django.views.generic.View` class and subclasses.
+     *
+     * See
+     *  - https://docs.djangoproject.com/en/3.1/topics/class-based-views/
+     *  - https://docs.djangoproject.com/en/3.1/ref/class-based-views/
+     */
+    module View {
+      /**
+       * An `API::Node` for references to `django.views.generic.View` or any subclass
```

Comment:
```suggestion
       * An `API::Node` representing the `django.views.generic.View` class or any subclass
```
The use of "references" might lead one to think that these give all of the corresponding `DataFlow::Node`s (that one would actually extract using `getAUse`). Would the above be a more fitting formulation?

Reply:
I like your reformulation better :muscle: